### ISSUE TITLE
Dependency missing when trying to use rabbitmq_federation plugin

### ISSUE THREAD

**Issue by masterphenix** [bitnami/bitnami-docker-rabbitmq#185]
**Description**

Hi there, we have a Kubernetes RabbitMQ cluster that is using image "docker.io/bitnami/rabbitmq:3.9.11".
We would like to enable rabbitmq_federation plugin, but activating it fails in regard to a dependency to 'rabbitmq_queue_master_balancer' that seems to be missing.

**Steps to reproduce the issue:**

1. Create a RabbitMQ server using the latest image 'docker.io/bitnami/rabbitmq:3.9.11' (we are using the RabbitMQ Cluster Operator to do that)
2. Exec into the 'rabbitmq' container (bash)
3. Execute following command : rabbitmq-plugins enable rabbitmq_federation

**Describe the results you received:**
```
Enabling plugins on node rabbit@sandbox-server-0.sandbox-nodes.rabbitmq-system:
rabbitmq_federation
Error:
{:plugins_not_found, [:rabbitmq_queue_master_balancer]}
```

**Describe the results you expected:**

I expected to be able to install rabbitmq_federation and rabbitmq_federation_management plugins with its dependencies.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Version**

Rabbitmq nodes are running on image 'docker.io/bitnami/rabbitmq:3.9.11', inside an AKS (Azure K8S) cluster, in version 1.21.2.

**Comment by juan131** [comment]
Hi @masterphenix 

As it's explained in the [REAME.md](https://github.com/bitnami/bitnami-docker-rabbitmq#node-and-cluster-configuration) you can set the env. variables `RABBITMQ_PLUGINS` and `RABBITMQ_COMMUNITY_PLUGINS` in the RabbitMQ container to indicate the plugins to install/enable during the container initialization, see:

* `RABBITMQ_PLUGINS`: Comma, semi-colon or space separated list of plugins to enable during the initialization. No defaults.
* `RABBITMQ_COMMUNITY_PLUGINS`: Comma, semi-colon or space separated list of URLs where to download custom plugins during the initialization. No defaults.

Could you give that a try?

**Comment by masterphenix** [comment]
Hello,
My bad, I was under the impression that installing the plugin rabbitmq_federation required a dependency that was not present in the image, which didn't seem right to me.

However, I checked my RabbitmqCluster resource, and it turns out that I was trying explicitely to enable the rabbitmq_queue_master_balancer plugin, by using standard plugins (RABBITMQ_PLUGINS) instead of community plugins.

I have moved rabbitmq_queue_master_balancer from RABBITMQ_PLUGINS to RABBITMQ_COMMUNITY_PLUGINS, and now everything is fine.
